task-driver/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  party0:
+    image: task-driver-integration-test:latest
+    build: .
+    ports:
+      - "8000:8000"
+    command: >
+      cargo test --test integration --
+        --party 0 
+        --port1 8000 
+        --port2 9000 
```

Comment:
does this mean we also need to publish port 9000?

Reply:
good catch, copied over the configuration in `circuits` and forgot to change it, updated